package.json

```diff
@@ -105,6 +105,7 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/preact": "^2.0.1",
     "@testing-library/preact-hooks": "^1.1.0",
+    "@testing-library/user-event": "^12.6.0",
```

Comment:
I added this to test tab key press.

Reply:
We had this initially for some other tests but it wasn't working so ended up using fireEvent. Good to see this is working for these tests.